camayoc/tests/qpc/cli/test_sources.py

```diff
@@ -72,8 +72,10 @@
 
 
 def generate_show_output(data):
-    """Generate a regex pattern with the data for a qpc cred show output."""
+    """Generate a regex pattern that matches `qpc source show` output (API v2)."""
     output = r"{\r\n"
+
+    # ─ credentials ─
```

Comment:
Also, I noticed that these comments are using "─" characters which looked odd to me, not like the typical `-` minus/hypthen character on our keyboards, and I had to copy and search to identify this character as [Box Drawings Light Horizontal (U+2500)](https://unicodeplus.com/U+2500). Were these comments copied out some other text editor, or did you enter these yourself?

Although I think that having "extended" high unicode characters in code is _technically_ okay, I think we should avoid using them unless we have a good reason.

Reply:
Good catch, and excellent advice!